build/system.gyp

```diff
@@ -35,6 +35,26 @@
        ],
       ],
     },
+    {
+      'target_name': 'libnotify',
+      'type': 'none',
+      'conditions': [
+        ['building_crosswalk_bin==1', {
+          'direct_dependent_settings': {
+            'defines': ['USE_LIBNOTIFY=1'],
+            'cflags': [
+              '<!@(pkg-config --cflags libnotify)',
+            ],
+          },
+          'link_settings': {
+	    'libraries': [
```

Comment:
The indentation is still missing one space, and you still need an `ldflags` section.